lib/turbo_tests/runner.rb

```diff
@@ -160,7 +160,7 @@ def start_subprocess(env, extra_args, tests, process_id, record_runtime:)
         command = [
           *command_name,
           *extra_args,
-          "--seed", rand(0xFFFF).to_s,
+          "--seed", @seed,
```

Comment:
`turbo_tests` still randomizes the value which is confusing. (My mistake, I'm sorry about that.)

Let's also change `lib/turbo_tests/runner.rb` so it doesn't add `seed` option by default:

https://github.com/serpapi/turbo_tests/blob/bb814e1a65ea5cf467e83a9d7aacb093802a5c82/lib/turbo_tests/runner.rb#L23

I'll try to finish #48 this week so it will also resolve what #57 is fixes.